Tasks/PackerBuildV1/DefaultTemplates/custom.linux.template.json

```diff
@@ -55,7 +55,7 @@
             "cd /deployTemp",
             "ls",
             "chmod +x /deployTemp/{{user `package_name`}}/{{user `script_relative_path`}}",
-            ". /deployTemp/{{user `package_name`}}/{{user `script_relative_path`}} {{user `script_arguments`}}"
```

Comment:
@maniSbindra 👋 please get into a time capsule and go couple of year back! 

I vaguely remember there was some case where script execution was not working without dot sourcing. If you could recall that, we can check that scenario to ensure nothing gets broken.

Reply:
hi @bishal-pdMSFT : This link should help https://github.com/microsoft/azure-pipelines-tasks/pull/7581#discussion_r206857746 . It Think this may have been due to a bug in  packer version 1.2.4.

Reply:
Thanks @maniSbindra. It is working fine for me even after removing dot. And I am not able to think of situation which would require us to have dot sourcing.
So, lets go ahead with this fix. I would see if customer report problems, and then revert if needed.